recipes/mdal/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+if [ "$(uname)" == "Linux" ]; then
+   export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+fi
+
+mkdir build
+cd build
+
+cmake -G "Unix Makefiles" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_LIBRARY_PATH=$PREFIX/lib \
+  -DCMAKE_INCLUDE_PATH=$PREFIX/include \
+  -DENABLE_TESTS=OFF \
+  ..
+
+# CircleCI offers two cores.
```

Comment:
This won't be build with Circle CI (IIRC Azure also has two cores assigned, but regardless, the variable will handle it)
```suggestion
```